debian/source/format

```diff
@@ -1 +1 @@
-3.0 (quilt)
+3.0 (native)
```

Comment:
Also for here, using mostly the quilt with my test script.

On native it have:
- *dpkg-source: Warnung: native Paketversion darf keine Revision haben*

Makes this maybe sense to separate e.g. between Kodi Matrix and Leia, as there e.g. "kodi-screensaver-shadertoy_3.0.2-0~focal.tar.xz" becomes created instead of "./kodi-screensaver-shadertoy_3.0.2.orig.tar.gz" where comes with "quilt"?

Reply:
it's just a warning, that launchpad ignores. Not using quilt makes it much easier to maintain the addon for multi distro series, since you have a tarball for each series, instead of having to use the same tarball for all builds.
So it's easier to update a single ubuntu series, that has failed to build  for whatever reason